src/shared/fstab-util.h

```diff
@@ -31,3 +31,7 @@ static inline bool fstab_test_yes_no_option(const char *opts, const char *yes_no
 }
 
 char *fstab_node_to_udev_node(const char *p);
+
+_pure_ static inline const char* fstab_path(void) {
```

Comment:
sounds unnecessary to declare this as pure... gcc should figure this out on its own in all cases, the function is after all inline...